src/store/reducers/cart/cart.actions.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  CHANGE_QUANTITY_IN_THE_CART,
+} from './types'
+
+export const addToCart = (
+  id: string,
+  title: string,
+  price: number,
+  quantity: number = 1,
+  image: string
+) => {
+  return {
+    type: ADD_TO_CART,
+    id,
+    title,
+    price,
+    quantity,
+    image,
+  }
+}
+
+export const deleteFromCart = (id: string) => {
+  return {
+    type: DELETE_FROM_CART,
+    id,
+  }
+}
+
+export const changeItemQuantity = (id: string, quantity: number) => {
+  return {
+    type: CHANGE_QUANTITY_IN_THE_CART,
```

Comment:
It's not a rule, but it makes code much more intuitive if your action creator name (`changeItemQuantity`) is the same as the action type (`CHANGE_QUANTITY_IN_THE_CART`). At the moment they are different. I suggest to rename the action type.